icees_api/features/sql.py

```diff
@@ -547,16 +547,27 @@ def create_cohort_view(conn, table_name, cohort_features):
                 "feature_qualifier": {
                     "operator": value["operator"],
                     "value": value["value"],
+                } if "value" in value else {
+                    # value only has two possible keys, value key for one value, and values key for a list of values
+                    "operator": value["operator"],
+                    # format value["values"] list into ("value1", "value2", ...,) for SQL IN operator query
+                    "value": '("{}")'.format('\", \"'.join(value["values"])),
                 },
             }
             for key, value in cohort_features.items()
         ]
+
     if cohort_features:
+        # only add quotation marks to value if operator is =
         condition_str = " AND ".join(
                 "\"{}\" {} \"{}\"".format(
                     feature["feature_name"],
                     feature["feature_qualifier"]["operator"],
                     feature["feature_qualifier"]["value"],
+                ) if feature["feature_qualifier"]["operator"] == '=' else "\"{}\" {} {}".format(
```

Comment:
For this one, I find this to be a little more readable and less verbose:
```python
condition_str = " AND ".join(
    ("\"{}\" {} \"{}\"" if feature["feature_qualifier"]["operator"] == '=' else "\"{}\" {} {}").format(
        feature["feature_name"],
        feature["feature_qualifier"]["operator"],
        feature["feature_qualifier"]["value"],
    )
    for feature in cohort_features)
```

Reply:
Unfortunately this cleaner code solution does not work. For some reason, ```.format()``` does not apply if the operator is equal to ```=``` with this simplified code solution, so I have kept the current code. Let me know if you have further suggestions. Thanks